srcpkgs/ananicy/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'ananicy'
+pkgname=ananicy
+version=2.2.1
+revision=1
+wrksrc=Ananicy-${version}
+depends="bash schedtool python3"
```

Comment:
```suggestion
depends="schedtool python3"
```

Reply:
Well, not all use bash and base-minimal does not include it, so adding it enforces it's normal operation under any conditions.

Reply:
But where does it use bash? It's a python script that doesn't call bash.